components/application-authenticators/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -647,7 +646,7 @@ protected String sendRequest(String url, String accessToken)
 
         reader.close();
```

Comment:
It's best to do clean up operations in a ```finally``` block. If an exception is thrown, clean up won't happen unless it's done in the ```finally``` block.